generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts

```diff
@@ -11,6 +10,9 @@ import { CSRFService } from '../auth/csrf.service';
 import { AuthServerProvider } from '../auth/auth-jwt.service';
 <%_ } _%>
 
+import SockJS = require('sockjs-client');
```

Comment:
why not `import { SockJs } from 'sockjs-client'` or `import * as SockJs from 'sockjs-client'` or something similar?